src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -59,23 +59,22 @@ const AvatarGrid = () => {
 
   // FIXME: This is really bad, needs to be fixed ASAP
   const partygoersBySeat: WithId<User>[][] = [];
-  partygoers &&
-    partygoers.forEach((partygoer) => {
-      if (
-        !venueId ||
-        !partygoer?.data ||
-        partygoer.data[venueId] === undefined ||
-        partygoer.data[venueId].row === undefined ||
-        partygoer.data[venueId].column === undefined
-      )
-        return;
-      const row = partygoer.data[venueId].row || 0;
-      const column = partygoer.data[venueId].column || 0;
-      if (!(row in partygoersBySeat)) {
-        partygoersBySeat[row] = [];
-      }
-      partygoersBySeat[row][column] = partygoer;
-    });
+  recentWorldUsers.forEach((partygoer) => {
```

Comment:
in `Audience.tsx` the iteration parameter was renamed to `user` but here it's still `partygoer` lets keep it consistent.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1047/commits/9699ddedbbba153966a325fdec6d8714c224a10c